website/client/src/comp/about.js

```diff
@@ -1,14 +1,89 @@
 import React from "react";
+import eulogyLogo from "../assets/img/EQ-halfling-tombstone.png";
 
 // This is the project description.
-const projectDescription = "We've always honored our fallen. Our loved ones were given proper burial and proper eulogy. Yet, many of our friends are online today. And as we get older, some will no longer be with us.";
+const presentationLink = "https://docs.google.com/presentation/d/1S0enStuspSFOCA47sYQGOw54QvP8QmfZr2Ybn5IPQbA/edit?slide=id.g33e6dbe26d6_0_10#slide=id.g33e6dbe26d6_0_10"
+
+
+const backgroundStyle = {
+    position: "absolute",
+    backgroundImage: `url(${eulogyLogo})`,
+    backgroundSize: "cover",
+    backgroundPosition: "center",
+    height: "100vh",
+    width: "100vw",
+    
+};
+
+const dimmedBlackBackgroundStyle = {
+    position: "relative",
+    top: 0,
+    left: "25vw",
+    height: "100vh",
+    width: "50vw",
+    background: "linear-gradient(to right, rgba(255, 255, 255, 0.8), rgba(255, 255, 255, 0.6), rgba(255, 255, 255, 0.8))",
+    zIndex: 1
+};
+
+const textStyle = {
+    position: "absolute",
+    top: "8vh",
+    left: "50%",
+    transform: "translate(-50%,0%)",
+    color: "rgb(0,0,0)",
+    zIndex: 2,
+    maxWidth: "80vw",
+    padding: "1rem",
+    textAlign: "center",
+
+}
+
 function About() {
     return (
-        <div style={{ display: 'flex', flexDirection: 'column', justifyContent: 'flex-end', height: '20vh' }}>
-            <h1>About</h1>
-            <p>
-                {projectDescription}
-            </p>
+        <div style={backgroundStyle}>
+        
+            <div style = {dimmedBlackBackgroundStyle}> 
+            </div>
+            <div style={textStyle}>
+                    <h1 style={{ fontSize: '2rem' }}>Why?</h1>
+                        <p>
+                           Why did we make this project? Many everquest players enjoy custom quests that they either download or
+                           make, but what if any player could make a quest based on either a famous person or a loved one in a few clicks?
+                           Eulogy Quest seeks to make it easier for Everquest players to create their own custom quests, and base their 
+                           quests on either a famous person, or a player they know from the game.
+                        </p>
+                    <h1 style={{ fontSize: '2rem' }}>How?</h1>    
+                        <p>
+                            The website has 2 sections: Famous Person Quest and Honored One Quest.
```

Comment:
Think it could be neat if Famous Person Quest and Honored One Quest linked to the quest generation pages, something like with some adjusted styling like you've got below in "More Info":
```suggestion
                            The website has 2 sections: 
                            <Link to="/famous-person"> Famous Person Quest</Link> and 
                            <Link to="/honored-one"> Honored One Quest</Link>.
```